.github/workflows/release-please.yml

```diff
@@ -0,0 +1,20 @@
+on:
+  push:
+    branches:
+      - main
```

Comment:
Should this also be triggering on `tags: {}`?

Reply:
Nope, this workflow is just for creating the tag, once the tag is created, the usual release pipeline is used.